src/lib.rs

```diff
@@ -158,21 +158,8 @@
 )]
 #![cfg_attr(feature = "nightly", allow(unstable_features))]
 
-extern crate log;
-
-extern crate failure;
-
-extern crate rayon;
-
-extern crate memmap;
-
-extern crate tempdir;
-
-extern crate tempfile;
-
-extern crate positioned_io;
-
-extern crate serde;
+#[macro_use]
```

Comment:
I prefer importing the macros over standard imports, but that is something that can be done later on.